### DISCUSSION TITLE
Invert value from sensor

### DISCUSSION THREAD

**Discussion by xZise** [kalkih/mini-graph-card#923] [Ideas]
Hi, I have multiple values which I want to display in a bar chart. Everything seems to work, except for that fact that one value is actually inverted/negated. Would it be possible to add an attribute to the entities object to invert one value?

**Comment by ildar170975** [comment]
One value for one particular moment of time?)))

**Reply by xZise** [reply]
Actually invert/negate all values of an entity or more like sensor. I want to compare three sensors with each other and one of them reports the value the other way around (positive/negative swapped).

**Reply by ildar170975** [reply]
Check this: https://github.com/kalkih/mini-graph-card/issues/591
Probably what you need is a "-1" multiplier.

**Reply by magiva** [reply]
"multiplier" is not a function of mini-graph-card ?

**Reply by ildar170975** [reply]
Not sure if I understood your question.
Currently there is no `multiplier` option for `mini-graph-card`.

**Reply by magiva** [reply]
im in need of inverting my numbers and came across this post.
multiplying the entitiy value by -1 would be perfect but i dont find a function to do this as part of mini-graph-card.
i thought 'multiplier' was a suggested function, i didnt find that in the docs, or a way to perform this calc.
i didnt want to have to create a new entity for all the ones i need to apply a calculation change to.

**Reply by ildar170975** [reply]
Currently you have to create a template sensor.
For a future - https://github.com/kalkih/mini-graph-card/discussions/923#discussioncomment-5020354
